deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml

```diff
@@ -683,7 +683,7 @@ spec:
       #      - -c
       #      - >-
       #        mkdir -p /etc/elastic-agent/inputs.d &&
-      #        wget -O - https://github.com/elastic/elastic-agent/archive/8.11.tar.gz | tar xz -C /etc/elastic-agent/inputs.d --strip=5 "elastic-agent-8.11/deploy/kubernetes/elastic-agent-standalone/templates.d"
+      #        wget -O - https://github.com/elastic/elastic-agent/archive/main.tar.gz | tar xz -C /etc/elastic-agent/inputs.d --strip=5 "elastic-agent-main/deploy/kubernetes/elastic-agent-standalone/templates.d"
```

Comment:
Something seems to be wrong with how these YAML files are templated, everything on the 8.11 branch references main which doesn't seem right.

Reply:
@cmacknz I had a small fix here: https://github.com/elastic/elastic-agent/pull/3267

So according to this,:
```
cd elastic-agent/deploy/kubernetes
make generate-k8s
```
And should have done it.

Reply:
@cmacknz found the reason, the [makefile](https://github.com/leehinman/elastic-agent/blob/894965abd22547f8d399e3721a76f36289fec1a5/deploy/kubernetes/Makefile#L4) reads the https://github.com/leehinman/elastic-agent/blob/894965abd22547f8d399e3721a76f36289fec1a5/version/docs/version.asciidoc#L2

So is this expected is not it?

Reply:
Nothing ever updates `doc-branch` in the agent repository since the docs are not published from this repository like they are in Beats. I suspect this file was originally copied from Beats without having the unnecessary parts removed.

The `stack-version` field is updated by our release automation, so you could use that instead. The only modification will be to strip the patch version if the intent is to download the latest version of the 8.11 branch. Using a tagged version would also work and probably be more stable, but I don't think that's what is happening today.